store/cards.ts

```diff
@@ -1,167 +1,88 @@
-    const manufacturer = get(card, 'manufacturer')
-    if (product) {
-      const sibling = products.find(
-        (product: Product) => product.code === card.code
-      )
-      handleSibling(card, sibling)
-    } else if (category) {
-      const sibling = findSiblingByNestedCode('categories', category)
-      handleSibling(card, sibling)
-    } else if (label) {
-      const sibling = findSiblingByNestedCode('productLabels', label)
-      handleSibling(card, sibling)
-    } else if (manufacturer) {
-      const sibling = products.find(
-        (product: Product) =>
-          product.manufacturer === manufacturer && !used.includes(product.code)
-      )
-      handleSibling(card, sibling)
-    }
-    return card
-  }
-}
-
-function bySortKey(a: Card, b: Card) {
-  return a.sortKey > b.sortKey ? 1 : b.sortKey > a.sortKey ? -1 : 0
-}
-
-const productsModule: Module<CardsState, any> = {
+const cardsModule: Module<CardsState, any> = {
   state(): CardsState {
     return {
-      used: [],
-      all: [
-        ['manufacturer', 'Coop'],
-        ['manufacturer', 'Oatly'],
-        ['manufacturer', 'Coop']
-        // ['product', '2317417300004'],
-        // ['product', '4008429006296'],
-        // ['product', '2317438900009'],
-        // ['category', '32484'], // Frukt och grönt
-        // ['category', '32484'], // Frukt och grönt
-        // ['category', '32484'], // Frukt och grönt
-        // ['category', '39033900'], // Vegetariskt
-        // ['label', 'KRAV0U0MARK'], // Kravmärkt
-        // ['label', 'KRAV0U0MARK'], // Kravmärkt
-        // ['label', 'alltidlagtpris'] // Alltid lågt pris
-      ].map(([key, value]) => {
-        const card: Card = {
-          name: 'Label',
-          sortKey: '',
-          type: CardTypes.INFO
+      extra: [
+        {
+          type: CardTypes.AD,
+          position: 12,
+          image: 'http://coop.se/img.webp',
+          link: 'http://coop.se/product'
+        },
+        {
+          type: CardTypes.INFO,
+          label: 'KRAV0U0MARK',
+          image: 'http://coop.se/img.webp',
+          imageAltText: '',
+          buttonText: '',
+          header: 'Label: Kravmärkt',
+          text: 'Ekologiskt betyder att det är...',
+          link: 'http://coop.se/product'
+        },
+        {
+          type: CardTypes.INFO,
+          brand: 'Änglamark',
+          image: 'http://coop.se/img.webp',
+          imageAltText: '',
+          buttonText: 'Mer info',
+          header: 'Brand: Änglamark',
+          text: 'Änglamark är vårt varumärke...',
+          link: 'http://coop.se/product'
         }
-        card.before = {}
-        if (key === 'product') {
-          card.before.product = value
+      ]
+    }
+  },
+  getters: {
+    getCards(state, getters, rootState, rootGetters) {
+      const cards = rootGetters['products/getProductsAsCards']
+      const used: string[] = []
+      state.extra.forEach((card) => {
+        if (card.type === CardTypes.PRODUCT) {
```

Comment:
product cards will never be an extra card

Reply:
Yeah, you're right, but typescript kept complaining so I just added it to quiet it for now.

Reply:
Done 👍